site/content/resources.md

```diff
@@ -96,6 +97,14 @@ This program provides primary care for uninsured patients, including immigrants
 
 To find local community health centers, search the Community Clinic Association of Los Angeles County.
 
+### Latino Health Resources
+
+[Latino Health Resource Guide](https://www.medicareadvantage.com/latino-health-resource-guide)
+
+[Latino Health Resource Guide - California](https://www.medicareadvantage.com/latino-health-resource-guide#CA)
```

Comment:
We can look into the resource guides and put the resources here instead of linking to another list? at least some of them or do both?